osmnx/geometries.py

```diff
@@ -335,11 +335,11 @@ def _create_gdf(response_jsons, polygon, tags):
         GeoDataFrame of geometries and their associated tags
     """
     # make sure we got data back from the server request(s)
-    element_count = sum([len(rj["elements"]) for rj in response_jsons])
+    element_count = sum(len(rj["elements"]) for rj in response_jsons)
     utils.log(f"{element_count} elements in the JSON responses (includes every node).")
 
     # if there are no elements in the responses
-    if element_count < 1:
+    if not element_count:
 
```

Comment:
Don't use implicit false with an integer value.

I wonder if this check can be simpler, like here: https://github.com/gboeing/osmnx/blob/551364776a06d9448ac0a379352182e91dd4817f/osmnx/graph.py#L534

Reply:
Definitely this would be better because it would just stop as soon as something is found. What about the `utils.log` line 339 that wants to get the number though?

Reply:
It looks like that log call could be moved to line 357 and merged with the exiting log call there. Something like `utils.log("Converting {count} elements in JSON responses to geometries")`

Reply:
Sounds good. I have pushed another commit, thank you for the hint.

The whole if-else block could be inverted (to have a positive condition) and anyway it is quite long and does not fit on a screen. But that would be a larger refactoring.

Reply:
Thanks, the summing of the list comprehension did bother me and so the `if not any()` looks like a neat alternative. Merging the log call makes sense as well.